spec/index.bs

```diff
@@ -935,6 +951,19 @@ and adds extra `[[`/`]]` in spec text. Consider removing the brackets.
         member corresponding to |key| in |device|.{{device/[[limits]]}} to the value |value|.
 </div>
 
+Any time the user agent needs to revoke access to a device, it calls
+[=lose the device=](device, `undefined`).
+
+<div algorithm>
+    To <dfn dfn>lose the device</dfn>(|device|, |reason|):
+
+    1. Set |device|.{{device/[[adapter]]}}.{{adapter/[[current]]}} to `false`.
+    1. Issue: explain how to get from |device| to its "primary" {{GPUDevice}}.
+    1. Resolve {{GPUDevice/lost|GPUDevice.lost}} with a new {{GPUDeviceLostInfo}} with
+        {{GPUDeviceLostInfo/reason}} set to |reason| and
```

Comment:
Doesn't need to be part of this change but, to minimize information disclosure, we should limit the information we provide developers in the `GPUDeviceReason` and `GPUDeviceLostInfo/reason` fields to only things they did wrong or some other ambiguous message.

**Good:** "You called the destroy method", "A validation error occurred because you screwed up", "The adapter was lost". 
**Bad:** "The user's laptop is now running on battery", "The user changed their power settings in the control panel."

Reply:
[GPUDeviceLostInfo.reason](https://gpuweb.github.io/gpuweb/#dom-gpudevicelostinfo-reason) is an enum type and can only be `undefined` or `"destroyed"`.

The `message`, though, is implementation-defined, so the concern is valid for it.